docs/running/server/wes.rst

```diff
@@ -79,28 +79,48 @@ Below is a detailed summary of all available options:
 .. _GA4GH docs on CORS: https://w3id.org/ga4gh/product-approval-support/cors
 
 
+.. _WESRunWithDockerCompose:
+
 Running the Server with `docker-compose`
 ----------------------------------------
 
 Instead of manually setting up the server components (``toil server``, RabbitMQ, and Celery), you can use the following
 ``docker-compose.yml`` file to orchestrate and link them together.
 
-Make sure to change ``/tmp/toil-workflows`` if you want Toil workflows to live somewhere else, and create the directory
+Make sure to change the credentials for basic authentication by updating the
+``traefik.http.middlewares.auth.basicauth.users`` label. (Note that single ``$`` signs need to be replaced with ``$$``
+in the yaml file).
```

Comment:
The user needs to use a tool to hash the password here, right?

Reply:
Correct